packages/integrations/node/test/fixtures/prerender-404/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@test/nodejs-prerender-404",
```

Comment:
Moved this from the `astro` package to the `@astrojs/node` package so we can test that the prerendered 404 page is properly handled in prod.